app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -924,4 +933,10 @@ private boolean bottomSheetHiddenOrCollapsed() {
         return sheetState == BottomSheetBehavior.STATE_HIDDEN
                 || sheetState == BottomSheetBehavior.STATE_COLLAPSED;
     }
+
+    private void openDonationPage() {
+        final Intent intent = new Intent(Intent.ACTION_VIEW,
+                Uri.parse(getString(R.string.donation_url)));
+        startActivity(intent);
+    }
```

Comment:
Use `ShareUtils` to create and start the intent.

Reply:
Done

Reply:
dejavu? didn't I fix that, did i? O.o

Reply:
I have this issue of fixing things and then forgetting to push. Did this happen again?

Reply:
Probably!